deployment/infra/k8s/event-store.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps/v1
+kind: StatefulSet                 # Type of Kubernetes resource
+metadata:
+  name: event-store            # Name of the Kubernetes resource
+spec:
+  replicas: 1                    # Number of pods to run at any given time
+  selector:
+    matchLabels:
+      app: event-store        # This deployment applies to any Pods matching the specified label
+  serviceName: event-store
+  template:                      # This deployment will create a set of pods using the configurations in this template
+    metadata:
+      labels:                    # The labels that will be applied to all of the pods in this deployment
+        app: event-store
+    spec:                        # Spec for the container which will run in the Pod
+      containers:
+        - image: eventstore/eventstore:22.10.1-alpha-arm64v8
```

Comment:
we need to find a way to specify the arch....

I also use openline cli on linux x86